postcss.config.js

```diff
@@ -0,0 +1,15 @@
+const autoprefixer = require('autoprefixer');
+const cssnano = require('cssnano');
+const mqpacker = require('css-mqpacker');
+
+const conf = require('./server/config');
+
+const options = {
+  plugins: [autoprefixer, mqpacker, cssnano]
+};
+
+if (conf.env === 'development') {
+  options.map = { inline: true };
```

Comment:
I only enabled sourcemaps in dev mode. We can adjust this as much as anybody cares... like maybe we dont want minification on the dev server, etc.